handlers.go

```diff
@@ -409,7 +409,7 @@ func (s *Server) handleStats(c echo.Context, u *util.User) error {
 	}
 
 	var contents []util.Content
-	if err := s.DB.Limit(limit).Offset(offset).Order("created_at desc").Find(&contents, "user_id = ? and active", u.ID).Error; err != nil {
+	if err := s.DB.Limit(limit).Offset(offset).Order("created_at desc").Find(&contents, "user_id = ?", u.ID).Error; err != nil {
```

Comment:
@kelindi can you create a separate table for active and inactive contents for the user? Create a tab here for active and inactive

![image](https://user-images.githubusercontent.com/4479171/196040497-bab14afc-70fe-45d1-a65f-1931d9def10d.png)